tests/openwisp2/settings.py

```diff
@@ -165,6 +172,14 @@
     OPENWISP_RADIUS_ORGANIZATIONRADIUSSETTINGS_MODEL = (
         'sample_radius.OrganizationRadiusSettings'
     )
+    # Rename sample_app database
+    DATABASES['default']['NAME'] = os.path.join(BASE_DIR, 'sample_radius.db')
```

Comment:
If you do 
```
SAMPLE_APP=1 python tests/manage.py migrate
./run-qa-checks
```
It fails because Migration order is disturbed, to avoid that, I've changed the name of the database! :smile: